Sources/Quick/Example.swift

```diff
@@ -141,6 +121,94 @@ final public class Example: _ExampleBase {
         }
         return aggregateFlags
     }
+
+    private func reportSkippedTest(_ testSkippedError: XCTSkip, name: String, callsite: Callsite) { // swiftlint:disable:this function_body_length
+        #if !canImport(Darwin)
+            return // This functionality is only supported by Apple's proprietary XCTest, not by swift-corelibs-xctest
+        #endif
+
+        guard let testRun = QuickSpec.current.testRun else { fatalError() } // FIXME: improve this
```

Comment:
I think keeping the guard is fine.

Let's just just `print()` and `return` like we do for the other guards below. 😊